packages/core/ai_content_platform/vimax/cli/commands.py

```
@@ -332,20 +332,63 @@ def generate_storyboard(script, output, image_model, style, portraits, reference
     async def run():
-        return await artist.process(script_obj, portrait_registry=portrait_registry)
-
-    result = run_async(run())
+        # Pre-resolve references from registry so CLI can display them before generation
+        if portrait_registry and len(portrait_registry.portraits) > 0:
+            resolved = await artist.resolve_references(script_obj, portrait_registry)
+            return resolved, await artist.process(script_obj, portrait_registry=portrait_registry)
+        # Inline refs or no refs — process directly
+        return 0, await artist.process(script_obj)
+
+    resolved_count, result = run_async(run())
```

Ruff (0.14.14)

[warning] Unpacked variable `resolved_count` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)